test/topology/test_mutation_schema_change.py

```diff
@@ -81,6 +82,7 @@ async def test_mutation_schema_change(manager, random_tables):
 
 
 @pytest.mark.asyncio
+@pytest.mark.enable_tablets(False)
```

Comment:
What is the long-term plan -- one we support LWT in tablets, how would we enable this test in tablets mode? Preferably, we should eventually run the test in both modes.

I'd prefer if this became more explicit in the code. Even the reason why you disable tablets here is not explicitly stated, there is no comment or anything, you have to look into the git log to find out.

I propose to take the approach that @bitpathfinder took in `test_change_replication_factor_1_to_0`, where the test is parameterized in advance to run in both tablets and vnodes mode, but the tablets mode is currently disabled, with explicit reason provided (issue reference):
```python3
@pytest.mark.parametrize(
    "use_tablets",
    [
        pytest.param(False, id="vnodes"),
        pytest.param(True, id="tablets", marks=pytest.mark.xfail(reason="issue #20282")),
    ],
)
@pytest.mark.asyncio
async def test_change_replication_factor_1_to_0(request: pytest.FixtureRequest, manager: ManagerClient, use_tablets: bool) -> None:
```

Reply:
It is interesting that this comment is not consistent with the previous one.  Some tests are parameterized on tablets, and some just use a CREATE KEYSPACE statement in them, and use the default. In fact, most of the tests are not parameterized. 

We could try to make sure that every test is explicitly parameterized to use tablets or vnodes. Oh wait. We have it. it's --enable-tablets in test/pylib/scylla_cluster.py. 

So if we have a way to parameterize all tests, do we need to separately parameterize each test? And moreover, do we need to do it *before* it actually works? Shouldn't the one who will be fixing LWT, CDC, views then actually test the fix anyway, including by enabling some of the disabled tests? Do we need to replace a grep + sed with pytest.mark.parameterize? Is there a guarantee that the bug will get closed? Is there a guarantee that when the bug is closed the test is updated? 

Too much to consider for such a simple problem. 
So the the question arises, why do you bother to ask me to do it this way and not the other way if there is so much ambiguity which way is better? 

Is it a good time and place to come up wit ha systematic way to parameterize the test? After all, the purpose and scope of the pull request is to prepare the ground for enabling tablets across the board, as, I hope, 
we will do eventually for all tests in test.py. 
The pull request has no aim to solve the larger problem of sequal testing of vnode-based and tablet-based partitioning, which should be solved for all tests.